--- ./stai-blockchain-gui/packages/wallets/src/components/WalletHeader.tsx	2022-03-08 03:20:47.225612000 -0800
+++ output2/./stai-blockchain-gui/packages/wallets/src/components/WalletHeader.tsx	2022-03-08 06:15:49.191164000 -0800
@@ -71,19 +71,6 @@
     <Flex gap={1} alignItems="center">
       <Flex flexGrow={1} gap={1}>
         <WalletsDropdodown walletId={walletId} />
-        <Button
-          color="primary"
-          onClick={handleAddToken}
-        >
-          <Trans>+ Add Token</Trans>
-        </Button>
-        <Button
-          color="primary"
-          variant="outlined"
-          onClick={handleManageOffers}
-        >
-          <Trans>Manage Offers</Trans>
-        </Button>
       </Flex>
       <Flex gap={1} alignItems="center">
         <Flex alignItems="center">
