--- ./stai-blockchain-gui/packages/wallets/src/components/WalletsList.tsx	2022-03-08 03:20:47.229612000 -0800
+++ output2/./stai-blockchain-gui/packages/wallets/src/components/WalletsList.tsx	2022-03-08 06:15:49.191164000 -0800
@@ -48,13 +48,6 @@
             ))}
           </>
         )}
-        <Grid xs={12} sm={6} md={4} item>
-          <WalletCreateCard
-            onSelect={handleAddToken}
-            title={<Trans>Add Token</Trans>}
-            icon={<AddIcon fontSize="large" color="primary" />}
-          />
-        </Grid>
       </Grid>
     </Flex>
   );
