--- ./stai-blockchain-gui/packages/gui/src/components/plot/add/PlotAddForm.tsx	2022-03-08 03:20:46.617644000 -0800
+++ output2/./stai-blockchain-gui/packages/gui/src/components/plot/add/PlotAddForm.tsx	2022-03-08 06:15:49.191164000 -0800
@@ -118,10 +118,7 @@
           initialTargetState,
           initialTargetState: { state },
         } = nftData;
-        const { transaction, p2SingletonPuzzleHash } = await createNewPoolWallet({
-          initialTargetState,
-          fee,
-        }).unwrap();
+        const { transaction, p2SingletonPuzzleHash } = await createNewPoolWallet(initialTargetState, fee).unwrap();
 
         if (!p2SingletonPuzzleHash) {
           throw new Error(t`p2SingletonPuzzleHash is not defined`);
@@ -186,7 +183,6 @@
           <PlotAddSelectTemporaryDirectory step={step++} plotter={plotter} />
         )}
         <PlotAddSelectFinalDirectory step={step++} plotter={plotter} />
-        <PlotAddNFT ref={addNFTref} step={step++} plotter={plotter} />
         <Flex gap={1}>
           <FormBackButton variant="outlined" />
           <ButtonLoading
