--- ./stai-blockchain-gui/packages/gui/src/components/app/AppSelectMode.tsx	2022-03-08 03:20:46.621643000 -0800
+++ output2/./stai-blockchain-gui/packages/gui/src/components/app/AppSelectMode.tsx	2022-03-08 06:15:49.191164000 -0800
@@ -94,18 +94,6 @@
                         <Trans>Store and Send STAI</Trans>
                       </Typography>
                     </Flex>
-                    <Flex gap={1} alignItems="center">
-                      <StyledCheckIcon color="primary" />
-                      <Typography variant="body2">
-                        <Trans>Manage CAT tokens</Trans>
-                      </Typography>
-                    </Flex>
-                    <Flex gap={1} alignItems="center">
-                      <StyledCheckIcon color="primary" />
-                      <Typography variant="body2">
-                        <Trans>Trade tokens</Trans>
-                      </Typography>
-                    </Flex>
                   </Flex>
 
                   <Button variant="outlined" fullWidth>
@@ -146,12 +134,6 @@
                       </Typography>
                     </Flex>
                     <Flex gap={1} alignItems="center">
-                      <StyledCheckIcon color="primary" />
-                      <Typography variant="body2">
-                        <Trans>Join farming pools</Trans>
-                      </Typography>
-                    </Flex>
-                    <Flex gap={1} alignItems="center">
                       <StyledCheckIcon color="primary" />
                       <Typography variant="body2">
                         <Trans>Runs the full node</Trans>
